contracts/README.md

```diff
@@ -140,7 +140,7 @@ $ yarn deploy:hardhat
 
 If the execution is successful, you will see the contract address on your screen.
 
-### **Polygon Mumbai Testnet**
+### **Ethereum Sepolia Testnet**
```

Comment:
Should we change entirely for Sepolia or we could keep both in docs as we already have run through mumbai?

Reply:
Ah good point, I vote for keeping it as well. I can change that.